app/services/clients/redivis.rb

```diff
@@ -3,17 +3,16 @@
 module Clients
   # Client for interacting with the Redivis API
   class Redivis < Clients::Base
-    def initialize(api_token:, organization:, url: 'https://redivis.com', conn: nil)
-      @organization = organization
+    def initialize(api_token:, url: 'https://redivis.com', conn: nil)
       super(url: url, api_token: api_token, conn: conn)
     end
 
     # @return [Array<Clients::ListResult>] array of ListResults for the datasets
     # @raise [Clients::Error] if the request fails
-    def list(max_results: 100)
-      results, page_token = list_page(max_results: max_results)
+    def list(organization:, max_results: 100)
```

Comment:
"organization" should be a method argument instead of an initializer argument since it is only used for this method.